rero_ils/modules/stats/views.py

```diff
@@ -80,6 +83,59 @@ def stats_librarian():
         type='librarian')
 
 
+@blueprint.route('/librarian/<record_pid>/csv')
+@check_logged_as_librarian
+def stats_librarian_queries(record_pid):
+    """Download specific statistic query into csv file."""
+    queries = {'1': 'loans_of_transaction_library_by_item_location'}
+    query_id = request.args.get('query_id', None)
+    if query_id not in queries:
```

Comment:
It seems that only `1` is allowed for the `query_id` parameter. If it is true this seems overkill.

Reply:
It was to foreseen the implementation of other queries in the future. Should I change it ?